src/Orbit/main.cpp

```diff
@@ -375,7 +376,11 @@ int main(int argc, char* argv[]) {
   command_line_flags =
       orbit_command_line_utils::RemoveFlagsNotPassedToMainWindow(command_line_flags);
 
+  auto confidence_interval_estimator =
```

Comment:
See the comments in App and below. I really think app should own the estimator.

Also, in general be very thoughtful about when to use smart pointers. The unique pointer will instantiate the estimator always on the heap. While otherwise, you might be able to stack allocate the objects. Also there is less indirections.